woeip/apps/air_quality/dustrak.py

```diff
@@ -237,20 +237,25 @@ def join(air_quality, gps, tolerance=3.):
     return joined_data
 
 
-def save(joined_data, session):
+def save(joined_data, gps_collection_file, pollutant_collection_file, pollutant):
     """Save a table of joined data to the database
 
     Parameters
     ----------
     joined_data : pandas DataFrame
-    session : woeip.apps.air_quality.models.Session
+    gps_collection_file : woeip.apps.air_quality.models.CollectionFile
+        CollectionFile object for GPS data
+    pollutant_collection_file : woeip.apps.air_quality.models.CollectionFile
+        CollectionFile object for pollutant data
+    pollutant : woeip.apps.air_quality.models.Pollutant
+        Pollutant that was collected
     """
-    data = []
     for _, row in joined_data.iterrows():
-        dat = models.Data(session=session,
-                          value=row['measurement'],
-                          time=row['time'],
-                          latlon=geos.Point(row['lon'], row['lat']))
-        data.append(dat)
-
-    models.Data.objects.bulk_create(data)
+        time_geo = models.TimeGeo(
+            collection_file=gps_collection_file, time=row['time'],
```

Comment:
Stylistic request: move `time` to its own row

Reply:
👍